includes/general.php

```diff
@@ -1108,19 +1109,24 @@ function pods_shortcode_run( $tags, $content = null ) {
 		return $return;
 	}//end if
 
+	$pagination = false;
+	if ( ! $is_singular && 0 < $found && $tags['pagination'] ) {
+		$pagination = array(
+			'label' => pods_v( 'pagination_label', $tags, null ),
+			'type'  => pods_v( 'pagination_type', $tags, null ),
+		);
+		// Remove empty params.
+		$pagination = array_filter( $pagination );
+	}
+
 	ob_start();
 
 	if ( ! $is_singular && false !== $tags['filters'] && 'before' === $tags['filters_location'] ) {
 		echo $pod->filters( $tags['filters'], $tags['filters_label'] );
 	}
 
-	if ( ! $is_singular && 0 < $found && true === $tags['pagination'] && in_array(
-		$tags['pagination_location'], array(
-			'before',
-			'both',
-		), true
-	) ) {
-		echo $pod->pagination( $tags['pagination_label'] );
+	if ( $tags['pagination'] && in_array( $tags['pagination_location'], array( 'before', 'both', ), true ) ) {
```

Comment:
```suggestion
	if ( $pagination && in_array( $tags['pagination_location'], array( 'before', 'both', ), true ) ) {
```

Reply:
This is invalid, `$pagination` could also be an empty array.